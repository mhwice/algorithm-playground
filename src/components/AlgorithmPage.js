import React from 'react';

const AlgorithmPage = () => (
	<div>
		AlgorithmPage
	</div>
);

export default AlgorithmPage;
