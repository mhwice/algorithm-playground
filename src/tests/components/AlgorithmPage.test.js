import React from 'react';
import { shallow } from 'enzyme';
import AlgorithmPage from './../../components/AlgorithmPage';

test('should render AlgorithmPage', () => {
  const wrapper = shallow(<AlgorithmPage />);
  expect(wrapper).toMatchSnapshot();
});