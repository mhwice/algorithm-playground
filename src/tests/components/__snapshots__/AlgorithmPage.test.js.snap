// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AlgorithmPage 1`] = `
<div>
  AlgorithmPage
</div>
`;
